rcl/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```

Comment:
> The xmllint is failing for rcl because I had to increase the package.xml version to 3 in order to add the group_depend and it doesn't seem to know how to deal with a version 3 file.

I think the xml schema referenced here `package_format2.xsd` will have to be changed or removed if the package format is bumped. Looks like there is a version 3 schema http://download.ros.org/schema/package_format3.xsd

Reply:
And the `group_depend` needs to be listed after the `test_depend`.

Reply:
Changed to format3 and moved the group_depend after the test_depend. Ran colcon test locally on Linux and it appears everything in `rcl` is passing.